node/pkg/cni/token_watch.go

```diff
@@ -244,7 +245,15 @@ func CNIServiceAccountName() string {
 		logrus.WithField("name", sa).Debug("Using service account from CALICO_CNI_SERVICE_ACCOUNT")
 		return sa
 	}
-	return defaultServiceAccountName
+	return defaultCNIPluginAccountName
+}
+
+func NodeServiceAccountName() string {
```

Comment:
I don't think we need this function - we shouldn't ever be doing any actions on the calico-node serviceaccount.